examples/animation/simple_anim.py

```diff
@@ -15,16 +15,27 @@
 line, = ax.plot(x, np.sin(x))
 
 
-def animate(i):
-    line.set_ydata(np.sin(x + i/10.0))  # update the data
+def init():  # only required for blitting to give a clean slate.
+    line.set_ydata(x)
```

Comment:
This sets the initial data to be a line?

Reply:
good catch, fixed